kernel_tuner/backends/cupy.py

```diff
@@ -124,6 +126,7 @@ def compile(self, kernel_instance):
         compiler_options = self.compiler_options
         if not any(["-std=" in opt for opt in self.compiler_options]):
             compiler_options = ["--std=c++11"] + self.compiler_options
+        # CuPy already sets the --gpu-architecture by itself, as per https://github.com/cupy/cupy/blob/20ccd63c0acc40969c851b1917dedeb032209e8b/cupy/cuda/compiler.py#L145
```

Comment:
Is there a recommend version of CuPy for KernelTuner? If so, consider replacing the link to e.g. `https://github.com/cupy/cupy/blob/v13/cupy/cuda/compiler.py#L145` for `v13` of CuPy. Something like https://pypi.org/project/bump2version/ may be helpful to keep such version numbers up to date going forward.

Reply:
Good point, it should indeed be version independent. I'll change it to `main` instead for now.